content/services.html

```diff
@@ -49,7 +49,8 @@ <h2>Online Things for Testing</h2>
 
     <p>
       Eclipse Thingweb hosts Things that can be interacted with over the public Internet.
-      You can use them to test your WoT applications.
+      You can use them to test your WoT applications. <br />
+      Note: we also provide a dedicated <a href="https://github.com/eclipse-thingweb/test-things/">GitHub repository</a> with a collection of IoT device simulators that can be used for testing and exploration purposes of different protocols and other Web of Things mechanisms.
```

Comment:
```suggestion
      Note: we also offer the <a href="https://github.com/eclipse-thingweb/test-things/">test-things</a> repository with a collection of IoT device simulators that can be used for testing and exploration purposes of different protocols and other Web of Things mechanisms.
```

Alternative?

Reply:
I like the alternative more

Reply:
Me too. Hence let's go with that proposal ...